src/status_im2/contexts/onboarding/welcome/view.cljs

```diff
@@ -40,10 +44,14 @@
        :icon-name  :i/arrow-left
        :on-press   #(rf/dispatch [:navigate-back-within-stack :enable-notifications])}]
      [page-title]
-     [rn/view {:style style/page-illustration}
-      [quo/text
-       "Illustration here"]]
+     [rn/image
+      {:style  (style/page-illustration (:width window))
+       :source (resources/get-image :welcome-illustration)}]
      [rn/view {:style (style/buttons insets)}
+      (when (< (:height window) 700)
```

Comment:
we have a function `small-screen?` in `react-native.core`

Reply:
Cool, done